app/controllers/general.php

```diff
@@ -543,8 +543,16 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
             $response->addHeader('Strict-Transport-Security', 'max-age=' . (60 * 60 * 24 * 126)); // 126 days
         }
 
+        $defaultResponseFormat = APP_VERSION_STABLE;
+        if ($defaultResponseFormat !== 'UNKNOWN') {
+            $defaultResponseFormat = \explode('.', $defaultResponseFormat);
+            $defaultResponseFormat[2] = '0';
```

Comment:
We should probably be defensive here and make sure the array has the right number of elements.

Reply:
Updated, but this is hardcoded, not user input so don't think there is huge risk here